core/db/migrate/20150213094131_attach_shipments_to_order_address.rb

```diff
@@ -0,0 +1,10 @@
+class AttachShipmentsToOrderAddress < ActiveRecord::Migration
+  def up
+    Spree::Shipment.where(address: nil).each do |shipment|
```

Comment:
Agree! I was already looking for something faster but couldn't come up with you solution :) will improve this!